kubejs/startup_scripts/fluids.js

```diff
@@ -12,10 +12,12 @@ StartupEvents.registry("fluid", (event) => {
         .renderType("translucent");
 
     event.create(global.fluids.andesite_mixture).thinTexture(0x808582);
+    event.create(global.fluids.molten_lunium_nova).thickTexture(0x8affd8);
```

Comment:
Did you try the tinkers texture too? with the `molten_metal` function?

Reply:
mmmmm nah

Reply:
ok